### PR TITLE
Add CSV validation.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#1090

### CODE REVIEW

File: packages/import/test/ValidateCsv.spec.ts [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+`;
+
+const CSV_WITH_TOKEN_WRONG_TYPE = `ID,DESCRIPTION,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS,TOKEN TYPE,TOKEN ID,TOKEN ISSUANCE
+programming_music.jpg,Programming Music,CC4.0,BY,wrongType,0x900edc98db53508e6742723988b872dd08cd09c2,1
+lucas_games_characters.jpg,LucasArts Games Characters,CC4.0,BY-NC,wrongType,0x900edc98db53508e6742723988b872dd08cd09c2,1
+`;
+
+
+
+fdescribe("ValidateCsv - readItemsCsv", () => {
```
Comment (gdethier) [inline packages/import/test/ValidateCsv.spec.ts]: There is a forgotten `fdescribe` here. Furthermore, I think that these "advanced" validation tests, along with their resources, should be moved to the `csv` package.

File: packages/import/test/ValidateCsv.spec.ts [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            expect(items[1].fileSize).toBe("91880");
+            expect(items[1].fileHash).toEqual(Hash.fromHex("0x546b3a31d340681f4c80d84ab317bbd85870e340d3c2feb24d0aceddf6f2fd31"));
+            expect(items[1].termsAndConditionsType).toBe("CC4.0");
+            expect(items[1].termsAndConditionsParameters).toBe("BY-NC");
+        } else {
+            fail("items expected in result");
+        }
+    });
+
+    fit("reads items with token with wrong token", async () => {
```
Comment (gdethier) [inline packages/import/test/ValidateCsv.spec.ts]: Forgotten `fit`.

**Review by gdethier — APPROVED** [review]
Great job! I left some comments, the one about moving tests can be addressed later.

File: packages/import/test/ValidateCsv.spec.ts (hunk 1 above)
Comment (benoitdevos) [reply packages/import/test/ValidateCsv.spec.ts]: I agree that `CsvValidator` should be unit-tested in its own package, although it way easier to setup tests data formatted in CSV. I'll fix that in a new PR.

File: packages/import/test/ValidateCsv.spec.ts (hunk 2 above)
Comment (benoitdevos) [reply packages/import/test/ValidateCsv.spec.ts]: I have force-pushed the forgotten `fit`and `fdescribe`.

**Pull request merged**